tests/test_order.py

```diff
@@ -0,0 +1,243 @@
+import os
+
+import pytest
+
+# pylint: disable=unused-import
+from .context import Order, Asset
+from .fixtures import (
+    ASSET_ID,
+    JSON_ORDER,
+    ORDER_ID,
+    WORKSPACE_ID,
+    auth_mock,
+    auth_live,
+    order_mock,
+    order_live,
+    asset_mock,
+    asset_live,
+)
+
+
+def test_init(order_mock):
+    assert isinstance(order_mock, Order)
+    assert order_mock.order_id == ORDER_ID
+    assert order_mock.workspace_id == WORKSPACE_ID
+
```

Comment:
Is this testing something specific? Or just to be extra extra cautious that parameters are getting passed correctly?

Reply:
Essentially this is testing if the __init__ method of the Order class is working as expected - i.e. that the object is initialized in the fixture and that the main attributes are defined correctly.